app/screens/about/about.js

```diff
@@ -228,11 +236,34 @@ export default class About extends PureComponent {
                         <FormattedText
                             id='mobile.about.copyright'
                             defaultMessage='Copyright 2015-{currentYear} Mattermost, Inc. All rights reserved'
-                            style={style.footerText}
+                            style={[style.footerText, {marginBottom: 0}]}
                             values={{
                                 currentYear: new Date().getFullYear(),
                             }}
                         />
+                        <FormattedText
+                            id='about.tos_privacy_text'
+                            defaultMessage='{tos} - {privacy}'
+                            style={style.footerText}
+                            values={{
+                                tos: (
+                                    <FormattedText
```

Comment:
These links are actually optional in the config.json, and if they're missing, we should hide the links.